service/history/consts/const.go

```diff
@@ -92,6 +92,8 @@ var (
 	ErrWorkflowTaskNotScheduled = serviceerror.NewWorkflowNotReady("Workflow task is not scheduled yet.")
 	// ErrNamespaceHandover is error indicating namespace is in handover state and cannot process request.
 	ErrNamespaceHandover = common.ErrNamespaceHandover
+	// ErrWorkflowTaskStateInconsistent is error indicating workflow task state is inconsistent, for example there was no workflow task scheduled but buffered events are present.
+	ErrWorkflowTaskStateInconsistent = serviceerror.NewUnavailable("Workflow task state is inconsistent.")
```

Comment:
Internal error maybe? I don't think retry can help in that inconsistent state?

Reply:
I think retry could help. WT can complete, buffered events might get flushed and maybe something else. Would leave it as `Unavailable` for now.